Project/OOP/Cylinder

```diff
@@ -1 +1,30 @@
-
+class Cylinder:
+    @staticmethod
+    def make_area(d, h):
+        circle = pi * d ** 2 / 4
+        side = pi * d * h
+        return round(circle*2 + side, 2)
+    
+    def __init__(self, diameter, high):
```

Comment:
btw, since you're familiar with c++/c#, try adding type annotations
https://docs.python.org/3/library/typing.html
https://towardsdatascience.com/type-annotations-in-python-d90990b172dc